app/Presenters/TransactionDigest.php

```diff
@@ -159,28 +168,27 @@ public function activities()
    */
   protected function analyzeRevisions()
   {
+    $importTransaction = $this->associatedImportTransaction;
     $this->buildObjectTypesToIds();
 
-    if ($this->wasCreate()) {
+    if ($importTransaction !== null) {
+      if ($importTransaction->import_action === 'update') {
+        $this->action = 'updated via import';
+      } else {
+        $this->action = 'created via import';
+      }
+      $this->importType = $importTransaction->import_type;
+    } else if ($this->wasCreate()) {
       $this->action = 'created';
     } else if ($this->wasUpdate()) {
       $this->action = 'updated';
     } else if ($this->wasDelete()) {
       $this->action = 'deleted';
     }
 
-    if ($this->wasAudioImport()) {
```

Comment:
Can you delete these functions now (wasAudioImport, wasVideoImport, and wasItemsImport if nothing else is using them?

Reply:
thanks for reminding me about this. done!